lib/src/crypto/src/crypto.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:typed_data';
+
+import '../../error/error.dart';
+import '../../generated/platform_constants.dart';
+import '../../platform/platform.dart';
+import '../crypto.dart';
+
+/// Crypto related methods related to creating keys ([generateRandomKey]) and configuration
```

Comment:
Given the name of this class is literally `Crypto`, I think this commentary could be simplified and made less repetitive:

```suggestion
/// Utility methods for creating keys ([generateRandomKey]) and configuration
```